### DISCUSSION TITLE
How to make regular backups of a server with date as a suffix.

### DISCUSSION THREAD

**Discussion by Peterkal2112** [Tyrrrz/DiscordChatExporter#580] [General]
Hello,
I am wondering how to export the whole server and save them into a folder with server name and date as a suffix that the backup was made.

So I want to save the whole server with truncating date in a name of a folder as a suffix and do it via cronjob regularly (i have the command and it works) in some specified interval once or twice a day, but the folder name is the same and when the second backup starts it will overwrite the old one.

So some sort of a flag with date in a name would be a nice feature to be implemented to be able to recognize the date that the backup of that server was made.

Can this be made via some sort of workaround temporarily when/if this kind of feature will be implemented.

Thank you for your answers.

**Comment by 96-LB** [comment]
If I'm understanding your use case correctly, this should be possible with [filename templates](https://github.com/Tyrrrz/DiscordChatExporter/wiki/GUI%2C-CLI-and-Formats-explained#generating-the-filename-and-output-directory-dynamically) already.

**Reply by Peterkal2112** [reply]
Yes, the filename templates are useful, but they don't include any data about date...

there are only ID's and server names and channel names but there are not any date or something...

These filename templates are not changing and are always staying the same so if the backup would run twice it would overwrite it without creating a new folder...

**Reply by 96-LB** [reply]
`%a` and `%b` are used to query the "after" and "before" dates of the range you're exporting. If you're exporting the entire channel history, `%b` should default to the current day.

**Reply by Peterkal2112** [reply]
You are genius , this is exactly what I wanted. Thank you very much.
the "%G - %b" as a name of a folder did exactly what I wanted.
Thank You.
